src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random RANDOM = new Random();
+
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Please replace the 'new Random()' with the 'RANDOM' constant that you've already defined. This avoids creating a new Random object each time the method is called, which is a part of the checklist point about not creating several instances of the same class if you can use only one instance for your purposes.